mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
 
 ## Commandline-only Options
 
-Those settings cannot be configured in the configuration files.
+The following settings cannot be configured in the configuration files:
+
+`--help`, `-h`
+:   Show brief usage information.
 
 `--force`, `-f`
-:   Replace the output file if it already exists, when building an
-    image. By default when building an image and an output artifact
-    already exists `mkosi` will refuse operation. Specify this option
-    once to delete all build artifacts from a previous run before
-    re-building the image. If incremental builds are enabled,
-    specifying this option twice will ensure the intermediary
-    cache files are removed, too, before the re-build is initiated. If a
-    package cache is used (also see the **Files** section below),
-    specifying this option thrice will ensure the package cache is
-    removed too, before the re-build is initiated. For the `clean`
-    operation this option has a slightly different effect: by default
-    the verb will only remove build artifacts from a previous run, when
-    specified once the incremental cache files are deleted too, and when
-    specified twice the package cache is also removed.
+:   Replace the output file if it already exists, by default `mkosi`refuses to overwrite an existing image.
+    This option can be specified once, twice or trice to change the scope of artifact removals.
+    Specify once to delete all build artifacts from a previous run. Twice to include removal of the
+    intermediary package cache from incremental builds. Thrice to remove all build artifacts and the the full
+    package cache. See the **Files** section for more context and details.
+    If `mkosi` is followed by the `clean` verb, the `-f` option can only be specified twice, see the
+    description of the command line verbs above.
```

Comment:
This does make it shorter, but I think we shouldn't "Mario, your commandline option description is in another castle." and describe the version of clean in a separate paragraph here as well.

Reply:
I don't agree, I think it's better to make the man page shorter by referring to previously declared descriptions, when available. This principle can be applied to many repetitions throughout the document.  So it's rather important to continue on the same page going forward.